core/src/testUtils/groovy/nu/yona/server/test/CommonAssertions.groovy

```diff
@@ -18,22 +18,22 @@ class CommonAssertions extends Service
 	static final UUID_PATTERN = '[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}'
 	static final VPN_LOGIN_ID_PATTERN = "(?i)^$UUID_PATTERN\\\$[0-9]+\$"
 
-	static final PUBLIC_USER_PROPERTIES_APP_NOT_OPENED = ["mobileNumber", "creationTime", "_links"] as Set
-	static final PUBLIC_USER_PROPERTIES_APP_OPENED = PUBLIC_USER_PROPERTIES_APP_NOT_OPENED + ["appLastOpenedDate"] as Set
-	static final PRIVATE_USER_PROPERTIES_CREATED_ON_BUDDY_REQUEST = PUBLIC_USER_PROPERTIES_APP_NOT_OPENED + ["firstName", "lastName", "nickname", "yonaPassword"] as Set
-	static final PRIVATE_USER_PROPERTIES_NUM_TO_BE_CONFIRMED = PRIVATE_USER_PROPERTIES_CREATED_ON_BUDDY_REQUEST + ["appLastOpenedDate"] as Set
-	static final PRIVATE_USER_PROPERTIES_NUM_CONFIRMED_BEFORE_ACTIVITY = PRIVATE_USER_PROPERTIES_NUM_TO_BE_CONFIRMED + ["vpnProfile", "sslRootCertCN", "_embedded"] as Set
-	static final PRIVATE_USER_PROPERTIES_NUM_CONFIRMED_AFTER_ACTIVITY = PRIVATE_USER_PROPERTIES_NUM_CONFIRMED_BEFORE_ACTIVITY + ["lastMonitoredActivityDate"] as Set
-	static final BUDDY_USER_PROPERTIES = PUBLIC_USER_PROPERTIES_APP_OPENED + ["firstName", "lastName", "nickname"] as Set
+	static final BASIC_USER_PROPERTIES_APP_NOT_OPENED = ["mobileNumber", "creationTime", "_links"] as Set
```

Comment:
Maybe we should do a terminology check on "basic user", "own user", "private user", "user", "buddy user", "minimal user"

Reply:
In response to this comment, I have removed the term "public user". The terms "private user" and "private user data" are still there, but if this data is to be omitted, then the server code clearly says ``xxxWithoutPrivateData``. In the above list of variables, I feel ``BASIC_`` is appropriate, as these properties/links are the basic ones, available in every user entity.

Reply:
So:

1. "own user" and "buddy user" refer to permissions of the user (authorization)
2. "minimal user" = "user without private data" and "user with private data" = "private user" refer to how much of the user is returned as requested and are not related to authorization
3. "basic user" is the intersection of all variants of a user

Am I correct?

Reply:
The user entity occurs in two situations:
* Your own user, generally just called "user"
* The user entity of a buddy, generally called "buddy user"

This naming convention is applied here as well. Next, there is a set of basic properties/links that applies to each of the two user variations.
Would it be helpful if I put these notes as a comment above the constants?

Reply:
That would be helpful. I would also like that we use the same terms in the code everywhere, but that is maybe already the case with your last update.

Reply:
Indeed, I think that's the case already. I forgot to add the above comment. Will do that in a separate commit.

Reply:
Great! That clarifies.

One final question: Is "minimal user" equal to "user without private data"?

Reply:
The answer to that is in the class ``nu.yona.server.subscriptions.rest.UserController.UserResourceRepresentation``. There it defines ``MINIMAL`` as ``false`` for all three options:
* ``includeGeneralContent``
* ``includeOwnUserNumNotConfirmedContent``
* ``includeOwnUserNumConfirmedContent``

So this contains less than a buddy user, which has ``true`` for ``includeGeneralContent``. The minimal format is used when embedding a user in a message.

Reply:
Thanks for the explanation and your patience!

Reply:
No issue at all!